library/cartridge_manage_failover.py

```diff
@@ -39,6 +49,80 @@ def manage_failover(params):
     return ModuleRes(success=True, changed=True)
 
 
+def manage_failover_new(control_console, failover_params):
+    current_failover_params = control_console.eval('''
+        return require('cartridge').failover_get_params()
+    ''')
+
+    mode = failover_params['mode']
+
+    lua_params = [
+        'mode = "{}"'.format(mode)
+    ]
+
+    if mode == 'stateful':
+        state_provider = failover_params.get('state_provider')
+        if state_provider == 'stateboard':
+            lua_params.append('state_provider = "tarantool"')
+
+            stateboard_params = failover_params.get('stateboard_params')
+
+            lua_stateboard_params = []
```

Comment:
It's just strange way to concat an arguments to string and eval then.

If you use tarantool-python connector you can form argument that you need in Python and then:
```py
arg = {"key": value}
conn.eval('''
    local arg = ...
    func(arg)
''', arg)
```

Reply:
I don't use `tarantool-python` connector, but I'm planning to improve passing arguments for Lua API calls.